### DISCUSSION TITLE
The ability to regex telemetry out of logs for graphing

### DISCUSSION THREAD

**Discussion by riaandotmodus** [grafana/grafana#38323] [Observability]
**What would you like to be added**:
I would like to be able to regex or jsonpath lines for use as metrics. E.g a json log line like:
{
   "outcome":{
      "value":"PERMIT",
      "isSuccess":true
   },
   "elapsedTime":**14665**
}

I would like to apply aggregates on the elapseTime in order to graph performance of the application

**Why is this needed**:
These logs come from a kubernetes environment where it would we are limited in what we can do with logs [referring to things like log mounts and sidecars] - aside from pushing them into loki with promtail or fluent-bit. The current features do not allow arbitrary values to be graphed and I do not see a way how to do this in a lightweight fashion

PS. Apologies if this is something that's doable already - I just really can't find anything in the docs regarding the how

**Comment by aocenas** [comment]
Is this related to any particular data source? Or do you think Grafana should do this regardless of the data source?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
